### PR TITLE
Adding .php_cs file as proposed by enebe-nb in #20 discussion

### PR DESCRIPTION
**Pull request by svpernova09**
Please discuss any changes here (if we need any before this is merged)

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+/*      // This must be used with 'header_comment', but then a config file is needed for each repository
+$header = <<<EOF
+Phergie (http://phergie.org)
+
+@link http://github.com/phergie/phergie-irc-bot-react for the canonical source repository
+@copyright Copyright (c) 2008-2015 Phergie Development Team (http://phergie.org)
```
Comment (elazar) [inline .php_cs]: Should the ending year be dynamic?

**Comment by svpernova09** [comment]
Not sure ```header_comment``` is working. I ran it a few times and it never seemed to change anything. Also unable to find any docs.

**Comment by enebe-nb** [comment]
I left the `header_comment` commented because it is a sugestion. The command `php-cs-fixer --dry-run --diff` will display the diffs without change any file, that will help to debug.

Now that i'm checking this, it only uses normal comment blocks to avoid conflict with doc blocks, but this way annotations won't work. Probably it won't help (although, the year is working).

**Comment by enebe-nb** [comment]
related to this: FriendsOfPHP/PHP-CS-Fixer#1297

**Comment by svpernova09** [comment]
Removed header. Once we finalize the Code Standards from #20 I'll update the .php_cs as needed and commit the changes files.

**Comment by svpernova09** [comment]
@enebe-nb Having some issues with this command on travis:

`output=$(php php-cs-fixer.phar fix -v --dry-run); if [[ $output ]]; then while read -r line; do echo -e "$line"; done <<< "$output"; false; fi;`

There's always output even if there are no changes. We need to grep for "begin diff" or something similar when there is something to fix.

**Comment by enebe-nb** [comment]
this command comes from my testing with that older version, it seem the exit code from cs-fixer changed many times. Probably using only `php php-cs-fixer.phar fix -v --dry-run` will work fine.

**Comment by enebe-nb** [comment]
Using the command directly works, but it gets a "Undefined Class" error on hhvm

**Comment by svpernova09** [comment]
Yeah I see, I'm not terribly worried about HHVM right now since we're still waiting for another issue with it to be resolved.

Looks good, thanks for the work on this!

**Pull request merged**